Makefile

```diff
@@ -0,0 +1,30 @@
+LOG_LEVEL ?= info
+
+build-docker:
+	chmod +x ./docker/build.sh && ./docker/build.sh
+
+mount-volume:
+	chmod +x ./docker/setup-volume.sh && ./docker/setup-volume.sh "$(CONFIG_PATH)"
+
+help:
+	@echo "Makefile commands:"
+	@echo "build-docker      Builds the Docker image."
+	@echo "mount-volume      Mounts the specified volume."
+	@echo "run               Runs the Docker container."
+	@echo "run-interactive   Runs the Docker container in interactive mode."
+	@echo "stop              Stops the Docker container."
+	@echo ""
+	@echo "Variables:"
+	@echo "CONFIG_PATH       Path to the config.json file."
+	@echo "LOG_LEVEL         Set the logging level (default: info) e.g. <make run LOG_LEVEL=debug>"
+	@echo "HELP              Set to true to print the help message (default: false) e.g. <make run HELP=true>"
+	@echo "PRINT_BATCH_SIZE  Set the batch size for printing the results (default: 1000) e.g. <make run PRINT_BATCH_SIZE=100>"
+
+run:
```

Comment:
nit: make these `docker-run` and `docker-stop`, since we may want to use `run` for `cargo run`.